cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,10 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('input[type="email"]').type('sofi@gmail.com');
+    cy.get('input[type="password"]').type('Sofi06!');
+    cy.get('button[type="submit"]').click();
+    cy.get('.nav-link').should('contain', 'sofi');
```

Comment:
It's a good practice to always add a timeout to your assertions. This is because sometimes, it might take a while for the page to load or for an element to be visible. By adding a timeout, you're telling Cypress to wait for a certain amount of time before failing the test. Here's how you can add a timeout:
```js
cy.get('.nav-link', { timeout: 10000 }).should('contain', 'sofi');
```